### ISSUE TITLE
all inputs trigger input event half the time

### ISSUE THREAD

**Issue by comhon-project** [formkit/formkit#463]
### Reproduction

[playground](https://formkit.com/playground?fkv=1.0.0-beta.11&fileTab=Playground.vue&files=JTVCJTdCJTIybmFtZSUyMiUzQSUyMlBsYXlncm91bmQudnVlJTIyJTJDJTIyZWRpdG9yJTIyJTNBJTIyJTNDc2NyaXB0JTIwc2V0dXAlM0UlNUNuaW1wb3J0JTIwJTdCJTIwcmVmJTIwJTdEJTIwZnJvbSUyMCd2dWUnJTVDbiUyRiUyRiUyMEVTTSUyMGltcG9ydHMlMjBhcmUlMjBzdXBwb3J0ZWQlMjBmcm9tJTIwVVJMcyUyMChodHRwcyUzQSUyRiUyRmNkbi4uLiklNUNuJTJGJTJGJTIwYXMlMjB3ZWxsJTIwYXMlMjBzZWNvbmRhcnklMjBwbGF5Z3JvdW5kJTIwZmlsZXMlMjAoLiUyRk90aGVyRmlsZS52dWUpLiU1Q24lNUNuY29uc3QlMjB0ZXN0JTIwJTNEJTIwcmVmKCU3QiU1Q24lMjAlMjB0ZXN0XzElM0ElMjAndmFsdWVfMSclMkMlNUNuJTIwJTIwdGVzdF8yJTNBJTIwJ3ZhbHVlXzInJTJDJTVDbiUyMCUyMHRlc3RfMyUzQSUyMCd2YWx1ZV8zJyUyQyU1Q24lMjAlMjB0ZXN0XzQlM0ElMjAndmFsdWVfNCclMkMlMjAlNUNuJTdEKSUzQiU1Q24lNUNuZnVuY3Rpb24lMjBpbnB1dChuYW1lKSUyMCU3QiU1Q24lMjAlMjBjb25zb2xlLmxvZygndXBkYXRlJyUyQyUyMG5hbWUpJTNCJTVDbiU3RCU1Q24lM0MlMkZzY3JpcHQlM0UlNUNuJTVDbiUzQ3RlbXBsYXRlJTNFJTVDbiUzQ0Zvcm1LaXQlMjB2LW1vZGVsJTNEJTVDJTIydGVzdCU1QyUyMiUyMHR5cGUlM0QlNUMlMjJmb3JtJTVDJTIyJTNFJTVDbiUyMCUyMCUzQ0Zvcm1LaXQlNUNuJTIwJTIwJTIwJTIwdHlwZSUzRCU1QyUyMnRleHQlNUMlMjIlNUNuJTIwJTIwJTIwJTIwbmFtZSUzRCU1QyUyMnRlc3RfMSU1QyUyMiU1Q24lMjAlMjAlMjAlMjBsYWJlbCUzRCU1QyUyMkZvcm1LaXQlMjBJbnB1dCU1QyUyMiU1Q24lMjAlMjAlMjAlMjBoZWxwJTNEJTVDJTIyZWRpdCUyMG1lJTIwdG8lMjBnZXQlMjBzdGFydGVkJTVDJTIyJTVDbiUyMCUyMCUyMCUyMCU0MGlucHV0JTNEJTVDJTIyKCklMjAlM0QlM0UlMjBpbnB1dCgndGVzdF8xJyklNUMlMjIlNUNuJTIwJTIwJTJGJTNFJTVDbiUyMCUyMCUzQ0Zvcm1LaXQlNUNuJTIwJTIwJTIwJTIwdHlwZSUzRCU1QyUyMnRleHQlNUMlMjIlNUNuJTIwJTIwJTIwJTIwbmFtZSUzRCU1QyUyMnRlc3RfMiU1QyUyMiU1Q24lMjAlMjAlMjAlMjBsYWJlbCUzRCU1QyUyMkZvcm1LaXQlMjBJbnB1dCU1QyUyMiU1Q24lMjAlMjAlMjAlMjBoZWxwJTNEJTVDJTIyZWRpdCUyMG1lJTIwdG8lMjBnZXQlMjBzdGFydGVkJTVDJTIyJTVDbiUyMCUyMCUyMCUyMCU0MGlucHV0JTNEJTVDJTIyKCklMjAlM0QlM0UlMjBpbnB1dCgndGVzdF8yJyklNUMlMjIlNUNuJTIwJTIwJTJGJTNFJTVDbiUyMCUyMCUzQ0Zvcm1LaXQlNUNuJTIwJTIwJTIwJTIwdHlwZSUzRCU1QyUyMnRleHQlNUMlMjIlNUNuJTIwJTIwJTIwJTIwbmFtZSUzRCU1QyUyMnRlc3RfMyU1QyUyMiU1Q24lMjAlMjAlMjAlMjBsYWJlbCUzRCU1QyUyMkZvcm1LaXQlMjBJbnB1dCU1QyUyMiU1Q24lMjAlMjAlMjAlMjBoZWxwJTNEJTVDJTIyZWRpdCUyMG1lJTIwdG8lMjBnZXQlMjBzdGFydGVkJTVDJTIyJTVDbiUyMCUyMCUyMCUyMCU0MGlucHV0JTNEJTVDJTIyKCklMjAlM0QlM0UlMjBpbnB1dCgndGVzdF8zJyklNUMlMjIlNUNuJTIwJTIwJTJGJTNFJTVDbiUyMCUyMCUzQ0Zvcm1LaXQlNUNuJTIwJTIwJTIwJTIwdHlwZSUzRCU1QyUyMnRleHQlNUMlMjIlNUNuJTIwJTIwJTIwJTIwbmFtZSUzRCU1QyUyMnRlc3RfNCU1QyUyMiU1Q24lMjAlMjAlMjAlMjBsYWJlbCUzRCU1QyUyMkZvcm1LaXQlMjBJbnB1dCU1QyUyMiU1Q24lMjAlMjAlMjAlMjBoZWxwJTNEJTVDJTIyZWRpdCUyMG1lJTIwdG8lMjBnZXQlMjBzdGFydGVkJTVDJTIyJTVDbiUyMCUyMCUyMCUyMCU0MGlucHV0JTNEJTVDJTIyKCklMjAlM0QlM0UlMjBpbnB1dCgndGVzdF80JyklNUMlMjIlNUNuJTIwJTIwJTJGJTNFJTVDbiUyMCUyMCUzQyUyRkZvcm1LaXQlM0UlNUNuJTNDJTJGdGVtcGxhdGUlM0UlNUNuJTVDbiUzQ3N0eWxlJTIwc2NvcGVkJTNFJTVDbiUyRiolNUNudmFuaWxsYSUyMENTUyUyMGNhbiUyMGdvJTIwaGVyZS4lNUNuS2VlcCUyMHN0eWxlcyUyMHNjb3BlZCUyMHRvJTIwYXZvaWQlMjBtdWx0aXBsZSUyMGZpbGVzJTVDbm92ZXJ3cml0aW5nJTIwZWFjaCUyMG90aGVyJTIwaW4lMjB0aGUlMjByZW5kZXIlMjBvdXRwdXQuJTVDbiolMkYlNUNuJTNDJTJGc3R5bGUlM0UlNUNuJTIyJTJDJTIyYWRkZWQlMjIlM0F0cnVlJTdEJTVE)

### Describe the bug

bug: 
all inputs trigger input event half the time (open dev console to see logs)

expected:
 ̶s̶h̶o̶u̶l̶d̶ ̶o̶n̶l̶y̶ ̶t̶r̶i̶g̶g̶e̶r̶ ̶o̶n̶e̶ ̶i̶n̶p̶u̶t̶ ̶e̶v̶e̶n̶t̶ ̶f̶o̶r̶ ̶t̶h̶e̶ ̶c̶u̶r̶r̶e̶n̶t̶ ̶f̶o̶c̶u̶s̶e̶d̶ ̶e̶l̶e̶m̶e̶n̶t̶
edit: should either 
- trigger one input event for the current focused element (every time)
- trigger all input events (every time)

### Environment

• FormKit beta 11

thanks in advance :)

**Comment by GustavoFenilli** [comment]
This behaviours is because v-model pass down the value to its children input, so it triggers because of that.

Not sure why it only trigger all on half of inputs.

**Comment by MortenHe** [comment]
I would also aspect that only the current focused input would trigger the input event.

**Comment by andrew-boyd** [comment]
@MortenHe Are you experiencing this issue locally in a project or only in the FormKit playground? 

The reason I ask is that I've identified an issue with the Vue import in the playground that I don't fully understand yet but it's affecting this issue. Here's your playground with the only change being updating `import { ref } from 'vue'` to `import { ref } from './vue'` and I'm now getting expected behavior. When I change an input I see console logs only for that input.

https://formkit.link/5f726eed76be270c57a1482f82368a18

😓 It's a weird one, but wanted to confirm you were seeing this issue in a project outside of the FormKit playground in a way that prompted you to open this issue.

**Comment by MortenHe** [comment]
@andrew-boyd I had the issue in local project

**Comment by abompard** [comment]
I am also experiencing this locally, I reproduced it with this snippet:
```vue
<script setup lang="ts">
const onChange = (nr: string, value: string) => {
  console.log("Change to", nr, value);
};
</script>
<template>
  <FormKit type="form">
    <FormKit type="text" @input="(val) => { onChange('1', val); }" />
    <FormKit type="text" @input="(val) => { onChange('2', val); }" />
    <FormKit type="text" @input="(val) => { onChange('3', val); }" />
  </FormKit>
</template>
```

On load, the first two inputs emit an input event with an undefined value. But not the third.
It does not seem related to refs as I don't use them here.

**Comment by justin-schroeder** [comment]
This is fixed on the `@next` tag (future release 0.17.x). You can check it out in the playground by switching to the next version in the version dropdown. 👍
